diff -Nur firefox-52.1.2esr.bak/build/moz.configure/old.configure firefox-52.1.2esr/build/moz.configure/old.configure
--- firefox-52.1.2esr.bak/build/moz.configure/old.configure	2017-04-11 11:13:08.000000000 +0900
+++ firefox-52.1.2esr/build/moz.configure/old.configure	2017-07-18 16:06:36.314197182 +0900
@@ -31,7 +31,7 @@
     autoconf = autoconf[0] if autoconf else None
 
     for ac in (mozconfig_autoconf, autoconf, 'autoconf-2.13', 'autoconf2.13',
-               'autoconf213'):
+               'autoconf213', 'true'):
         if ac:
             autoconf = find_program(ac)
             if autoconf:
